app/api/margatsni/v1/users/auth.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Margatsni
+  module V1
+    module Users
+      module Auth
```

Comment:
This module definitely acts as a helper, so let's move this file to `app/api/margatsni/v1/helpers` folder